From 9046bbf05bd3530aa8edf247190693f533f0ca27 Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Wed, 1 May 2013 09:55:17 +0200
Subject: [PATCH 36/37] BUILD: stdbool is not portable (again)

Another build issue on Solaris without c99. Please don't use stdbool.
---
 src/checks.c                  | 1 -
 src/haproxy-systemd-wrapper.c | 1 -
 2 files changed, 2 deletions(-)

diff --git a/src/checks.c b/src/checks.c
index f1072cd..d1b7a36 100644
--- a/src/checks.c
+++ b/src/checks.c
@@ -25,7 +25,6 @@
 #include <netinet/in.h>
 #include <netinet/tcp.h>
 #include <arpa/inet.h>
-#include <stdbool.h>
 
 #include <common/chunk.h>
 #include <common/compat.h>
diff --git a/src/haproxy-systemd-wrapper.c b/src/haproxy-systemd-wrapper.c
index 8499b35..fb1a7fd 100644
--- a/src/haproxy-systemd-wrapper.c
+++ b/src/haproxy-systemd-wrapper.c
@@ -12,7 +12,6 @@
 
 #include <errno.h>
 #include <signal.h>
-#include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-- 
1.8.1.5

